### ISSUE TITLE
How to create Sub menu 

### ISSUE THREAD

**Issue by tanveerahmed1980** [leocb/MaterialSkin#293]
How can i create child menu inside parent menu lime 
Dashboard 
-Blank dash board
-dashboard with widget

**Comment by valimaties** [comment]
I don't think that `Drawer` items can have childs right now. It is interesting, but I Don't know how this could be implemented, as long as, now, the drawer is build from a `TabControl` object. 
Maybe, if a `TabPage` contains another `TabControl` without a `TabSelector` attached, this `TabControl` child could be attached in `Drawer` as childs for `TabPage`.
